woocommerce/woocommerce-ios
# Networking: Add Fees to Orders

This is the first step to add support for Fees to the Order Details page. (#3084)

This PR implements support for Fees in Networking. Support for Fees in Yosemite and Storage will be added in subsequent PRs.

## Changes
- Add new model objects to Networking: `OrderFeeLine`, `OrderFeeTaxStatus` 
- Update `Order` to add a new property called `fees`
- Update mocks and json payloads in tests target
- Add a test to OrderMapperTests to test the parsing of an individual Order

## How to test
There are no user facing changes. 
- 👀 at the code 
- Check for a ✅ on CI.

Update release notes:
